packages/apps/api/src/handlers/errorHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response } from 'express';
+import { HttpError } from '../models/HttpError';
+import { toMessage } from '../models/common';
+
+export const errorHandler = (
+  err: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  if (res.headersSent) {
+    return next(err);
+  }
+
+  const status = err instanceof HttpError ? err.status : 500;
+  const message = err.message ?? 'Something went wrong';
```

Comment:
Very minor: our approach elsewhere is to omit the message if there's nothing useful to say.